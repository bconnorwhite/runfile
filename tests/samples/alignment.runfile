# ----------
# Alignment Test
# ----------

# Short command
short
  a?  # Short arg
  --flag: # Short flag
  echo "short"

# Very long command name
verylongcommandname
  verylongargumentname?   # Very long argument name
  -s, --verylongflagname: # Very long flag name
  echo "very long"
