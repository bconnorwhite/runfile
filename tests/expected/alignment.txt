Alignment Test
  short                     # Short command
    a?                        # Short arg
    --flag                    # Short flag
  verylongcommandname       # Very long command name
    verylongargumentname?     # Very long argument name
    -s, --verylongflagname    # Very long flag name

